Kickstarter-iOS/Views/Controllers/LiveStreamCountdownViewController.swift

```diff
@@ -377,32 +379,3 @@ public final class LiveStreamCountdownViewController: UIViewController {
     self.eventDetailsViewModel.inputs.subscribeButtonTapped()
   }
 }
-
-// Returns a fancy monospaced font for the countdown.
-private func countdownFont(label: UILabel) -> UIFont {
```

Comment:
we use this twice now so i extracted it to a helper on `UIFont`, can just do `baseFont.countdownMonospaced`